public/server/db/imu.json

```diff
@@ -0,0 +1,4 @@
+{
+    "definition": "I Miss U",
```

Comment:
Seeing as the letter `U` in this case is a small abbreviation on its own, I'd rather we go with the alternative as the definition and leave it at `I Miss You`. What do you think?

Reply:
We can definitely go with the alternative.